### DISCUSSION TITLE
Puppeteer sample

### DISCUSSION THREAD

**Discussion by jonathangiber** [getgauge/gauge#1929] [General]
Hi guys,
Was wondering if there is any gauge sample with puppeteer out there?
Thanks

**Comment by sriv** [comment]
hi @jonathangiber - we do not have a puppeteer sample as part of the examples. Perhaps someone from the community have something to share?

**Comment by WallisonLima** [comment]
Olá, eu tenho uma amostra em meu repositório, ele demonstra um função básica do tireteiro de abrir o navegador e realizar o acesso em uma url varias vezes, abrindo sempre novas abas. Esta meio bagunçado... mas queira mais ajuda estarei aberto e disposto ajudar com o entendimento e estudo de novas funcionalidades. Espero ajudá-lo!
https://github.com/WallisonLima/Puppeteer

**Reply by sriv** [reply]
hi @WallisonLima 

I don't understand your language (Portugese, I guess), but google translate tells me you meant this:

> Hello, I have a sample in my repository, it demonstrates a basic function of the tireteiro to open the browser and perform the access in a url several times, always opening new tabs. It is a bit messy ... but want more help I will be open and willing to help with the understanding and study of new features. I hope to help you!

Thanks for the offer to help, we'd be glad to accept any contribution. If you wish to write an example or a blog post around how to use gauge+puppeteer please do share here!
